LICENSE

```diff
@@ -1,339 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```

Comment:
replace with Apache?

Reply:
Opted to keep the previous license instead, since I didn't need to remove this file in order to apply David's changes.